WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -342,7 +342,12 @@ extension SiteDesignContentCollectionViewController: CategorySectionTableViewCel
             completion: completion
         )
 
-        let navController = GutenbergLightNavigationController(rootViewController: previewVC)
+        let navController: UINavigationController
+        if #available(iOS 26, *) {
+            navController = UINavigationController(rootViewController: previewVC)
+        } else {
+            navController = GutenbergLightNavigationController(rootViewController: previewVC)
```

Comment:
This if check is duplicated in a few places. Maybe we can update `GutenbergLightNavigationController` to only execute the customizing appearance code on pre-iOS 26 devices?